public/app/dashboard/splash.html

```diff
@@ -2,15 +2,50 @@
 <div data-ng-include="'/app/dashboard/myactivity.html'"></div>
 
 <div data-ng-if="userActivity">
-  <h3 data-ng-bind="user.id === me.id ? 'My Activity' : 'Activity'"></h3>
+  <h3 data-ng-bind="isMyProfile ? 'My Activity' : 'Activity'"></h3>
   <div class="activity-timeline"
        data-activity-timeline="userActivity"
        data-labels-width="200"></div>
 </div>
 
 <div data-ng-if="userAssetActivity">
-  <h3 data-ng-bind="user.id === me.id ? 'My Classmates' : 'Classmates'"></h3>
+  <h3 data-ng-bind="isMyProfile ? 'My Classmates' : 'Classmates'"></h3>
   <div class="activity-timeline"
        data-activity-timeline="userAssetActivity"
        data-labels-width="200"></div>
 </div>
+
+<div class="featured-assets-container" data-ng-if="featuredAssets && featuredAssets.length > 0">
```

Comment:
@johncrossman In a template I think you're safe with just `data-ng-if="featuredAssets.length"`.